MPF.Frontend/Tools/SubmissionGenerator.cs

```diff
@@ -396,13 +400,41 @@ private static SubmissionInfo CreateDefaultSubmissionInfo(BaseProcessor processo
             return internalProgram.LongName();
         }
 
+        /// <summary>
+        /// Simplifies a volume label into uppercase alphanumeric only string
+        /// </summary>
+        /// <param name="labels">Volume label to simplify</param>
+        /// <returns>Simplified volume label</returns>
+        private static string? SimplifyVolumeLabel(string? label)
+        {
+            if (label == null || label.Length == 0)
+                return label;
+            
+            var labelBuilder = new StringBuilder();
+            foreach (char c in label)
+            {
+                if ((c >= 'A' && c <= 'Z') || (c >= 'a' && c <= 'z') || (c >= '0' && c <= '9'))
+                    labelBuilder.Append(c);
```

Comment:
I think there's an efficient `ToUpperInvariant` that can be used on characters here. It might be easier.

Reply:
I've chosen to specifically do this because I want ASCII only. Microsoft's functions are happy to pass Unicode variants

Reply:
There should also be a `char.ToUpper` that can be used.